kafka_enforcer_common/src/main/java/com/tesla/data/enforcer/KafkaAdminClientProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.tesla.data.enforcer;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+
+import java.util.Map;
+
+public class KafkaAdminClientProvider implements ClientProvider {
```

Comment:
I would expect the Kafka one to also provide a method to get the config block given a `Map<String, Object>` command config... or at least a config key.